src/Makefile

```diff
@@ -127,7 +127,7 @@ else
 ifeq ($(OS), Darwin)
 CG_LLVMLINK += $(LLVM_LDFLAGS) -lLLVM
 else
-CG_LLVMLINK += $(LLVM_LDFLAGS) -lLLVM-13jl
+CG_LLVMLINK += $(LLVM_LDFLAGS) -lLLVM-$(LLVM_VER_MAJ)jl
```

Comment:
@staticfloat the one line that concerns you xD

Reply:
I like it!

Reply:
Where did you get this? You are prohibited from this variable here, since it is in `deps`. This file uses `$(LLVM_CONFIG_HOST)`.

Reply:
```
vchuravy@odin ~/b/julia-llvm14> make print-LLVM_VER_MAJ -C src
make: Entering directory '/home/vchuravy/builds/julia-llvm14/src'
LLVM_VER_MAJ=14
make: Leaving directory '/home/vchuravy/builds/julia-llvm14/src'
```